data_model/src/transaction.rs

```diff
@@ -349,9 +369,9 @@ impl PartialOrd for TransactionValue {
 impl Ord for TransactionValue {
```

Comment:
sorry, I just noticed this is inconsistent. Remove `PartialOrd` and `Ord` manual impls and derive them